checkpoint/module.go

```diff
@@ -157,47 +155,48 @@ func (AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.Validato
 // Internal methods
 //
 func verifyGenesis(state types.GenesisState) error {
-	contractCaller, err := helper.NewContractCaller()
-	if err != nil {
-		return err
-	}
-
+	// contractCaller, err := helper.NewContractCaller()
+	// if err != nil {
+	// 	return err
+	// }
+	// TODO: figure out maybe fetch address from chainmanager genesis itself
```

Comment:
Fix it by using chain manager genesis state.

Reply:
how do I fetch `chainManager` genesis state into modules?